test/matchers/size_validator_matcher_test.rb

```diff
@@ -88,10 +88,19 @@ module OnlyMatchWhenExactValue
 
   describe '#greater_than' do
     let(:matcher_method) { :greater_than }
-    let(:model_attribute) { matcher_method }
     let(:validator_value) { 7.kilobytes }
 
-    include SizeValidatorMatcherTest::OnlyMatchWhenExactValue
+    describe 'with has_one_attached' do
+      let(:model_attribute) { matcher_method }
+
+      include SizeValidatorMatcherTest::OnlyMatchWhenExactValue
+    end
+
+    describe 'with has_many_attached' do
+      let(:model_attribute) { :many_greater_than }
+
+      include SizeValidatorMatcherTest::OnlyMatchWhenExactValue
+    end
```

Comment:
Better do something like:
```ruby
    let(:model_attribute) { matcher_method }
    let(:validator_value) { 7.kilobytes }

    describe 'with has_one_attached' do
      let(:matcher_method) { :greater_than }

      include SizeValidatorMatcherTest::OnlyMatchWhenExactValue
    end

    describe 'with has_many_attached' do
      let(:matcher_method) { :many_greater_than }

      include SizeValidatorMatcherTest::OnlyMatchWhenExactValue
    end
```

Reply:
I don't understand what you mean. Do you want me to implement a separate matcher method for `many_greater_than` in `SizeValidatorMatcher` which would be the same as `greater_than`?

Reply:
Done. I had to create an alias for `SizeValidatorMatcher#many_greater_than`.

Reply:
@yenshirak I think I made a mistake, my bad,your original code was the right one. We shouldntt have to add an alias in the `size_validator_matcher.rb` file.
Sorry for the inconvenience, could you revert this change?

Reply:
done

Reply:
Thanks a lot @yenshirak!